app/src/ui/layout/layout.css

```diff
@@ -32,6 +32,8 @@
   }
 }
 
+.scroll-y { overflow-y: scroll; }
+.pad-sm { padding: var(--pad-sm) }
```

Comment:
I don't like classnames that describe CSS (unless it's a framework); I much prefer classnames that describe the state. We should change these to be `is-scrollable` and `is-compact` to reflect the property names and state of those sections.